server/app/services/program/EligibilityDefinition.java

```diff
@@ -17,8 +20,15 @@
 @JsonDeserialize(builder = AutoValue_EligibilityDefinition.Builder.class)
 @AutoValue
 public abstract class EligibilityDefinition {
-  // Note: While there is only one member currently, future iterations will add more which requires
-  // a wrapping class.
+
+  /** Indicates the shape of the predicate's AST so view code can render the appropriate UI. */
+  public enum PredicateFormat {
+    // A single leaf node
+    SINGLE_QUESTION,
+    // A top level conjunction with only AND child nodes,
+    // each of which has only leaf nodes.
+    SINGLE_LAYER_AND;
```

Comment:
Suggestion: explicitly rename this OR_OF_ANDS ?

Single layer has an implicit OR, so better to be explicit.

Reply:
done, and moved to `PredicateDefinition`